docs/faq.rst

```diff
@@ -132,8 +132,10 @@ completed is useful because it references all the transmitted chunks up
 to the checkpoint. This means that in case of an interruption, you only need to
 retransfer the data since the last checkpoint.
 
-If a backup was interrupted, you do not need to do any special considerations,
-just invoke ``borg create`` as you always do. You may use the same archive name
+If a backup was interrupted, you normally do not need to do anything special,
+just invoke ``borg create`` as you always do. (If locks are still on a repository,
+such as if the client suffered a sudden poweroff during a backup, you may need to
+run ``borg break-lock`` before the next backup.) You may use the same archive name
```

Comment:
How about: "In case the repository is still locked, you may need to ..."

no outer parens. and we also do not need to speculate about the reasons why it is still locked.